Description: Change the prefix from /usr/local to /usr
 Change installation prefix to /usr
 .
 lumina-desktop (0.6.2.14-1nano) unstable; urgency=low
 .
   * Initial release
Author: Christopher Roy Bratusek <nano@jpberlin.de>

--- lumina-desktop-0.6.2.14.orig/libLumina/libLumina.pro
+++ lumina-desktop-0.6.2.14/libLumina/libLumina.pro
@@ -2,7 +2,7 @@
 QT       += core
 
 TARGET=LuminaUtils
-target.path = /usr/local/lib
+target.path = /usr/lib
 
 DESTDIR= $$_PRO_FILE_PWD_/
 
@@ -21,11 +21,11 @@ SOURCES	+= LuminaXDG.cpp \
 	LuminaOS-FreeBSD.cpp \
 	LuminaOS-Linux.cpp
 
-INCLUDEPATH += /usr/local/include
+INCLUDEPATH += /usr/include
 
 LIBS	+= -lX11 -lXrender -lXcomposite
 
-include.path=/usr/local/include/
+include.path=/usr/include/
 include.files=LuminaXDG.h \
 	LuminaUtils.h \
 	LuminaX11.h \
@@ -33,4 +33,4 @@ include.files=LuminaXDG.h \
 
 INSTALLS += target include 
 
-QMAKE_LIBDIR = /usr/local/lib/qt4 /usr/local/lib
+QMAKE_LIBDIR = /usr/lib/qt4 /usr/lib
--- lumina-desktop-0.6.2.14.orig/lumina-config/lumina-config.pro
+++ lumina-desktop-0.6.2.14/lumina-config/lumina-config.pro
@@ -2,7 +2,7 @@
 QT       += core gui
 
 TARGET = lumina-config
-target.path = /usr/local/bin
+target.path = /usr/bin
 
 TEMPLATE = app
 
@@ -20,7 +20,7 @@ FORMS    += mainUI.ui \
 
 # RESOURCES+= lumina-config.qrc
 
-INCLUDEPATH += ../libLumina /usr/local/include
+INCLUDEPATH += ../libLumina /usr/include
 
 linux-g++-64 {
   LIBS     += -L../libLumina -lLuminaUtils 
@@ -93,7 +93,7 @@ TRANSLATIONS =  i18n/lumina-config_af.ts
                 i18n/lumina-config_zh_TW.ts \
                 i18n/lumina-config_zu.ts
 
-dotrans.path=/usr/local/share/Lumina-DE/i18n/
-dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/local/share/Lumina-DE/i18n/
+dotrans.path=/usr/share/Lumina-DE/i18n/
+dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/share/Lumina-DE/i18n/
 
 INSTALLS += target dotrans
--- lumina-desktop-0.6.2.14.orig/lumina-desktop/lumina-desktop.pro
+++ lumina-desktop-0.6.2.14/lumina-desktop/lumina-desktop.pro
@@ -2,7 +2,7 @@
 QT       += core gui network phonon
 
 TARGET = Lumina-DE
-target.path = /usr/local/bin
+target.path = /usr/bin
 
 LIBS     += -L../libLumina -lLuminaUtils -lXdamage -lX11
 QMAKE_LIBDIR	= ../libLumina
@@ -78,25 +78,25 @@ FORMS    += SystemWindow.ui \
 
 RESOURCES+= Lumina-DE.qrc
 
-INCLUDEPATH += ../libLumina /usr/local/include
+INCLUDEPATH += ../libLumina /usr/include
 
 desktop.files = Lumina-DE.desktop
 linux-g++-64 {
 desktop.path = /usr/share/xsessions
 } else {
-desktop.path = /usr/local/share/xsessions
+desktop.path = /usr/share/xsessions
 }
 
 icons.files = Lumina-DE.png \
 		Insight-FileManager.png
 
-icons.path = /usr/local/share/pixmaps
+icons.path = /usr/share/pixmaps
 
 wallpapers.files = wallpapers/Lumina_Wispy_gold_1920x1080.jpg \
 			wallpapers/Lumina_Wispy_green_1920x1080.jpg \
 			wallpapers/Lumina_Wispy_purple_1920x1080.jpg \
 			wallpapers/Lumina_Wispy_red_1920x1080.jpg
-wallpapers.path = /usr/local/share/wallpapers/Lumina-DE
+wallpapers.path = /usr/share/wallpapers/Lumina-DE
 
 defaults.files = defaults/desktop-background.jpg \
 		defaults/defaultapps.conf \
@@ -104,7 +104,7 @@ defaults.files = defaults/desktop-backgr
 		defaults/stylesheet.qss \
 		audiofiles/Logout.ogg \
 		audiofiles/Login.ogg
-defaults.path = /usr/local/share/Lumina-DE/
+defaults.path = /usr/share/Lumina-DE/
 
 TRANSLATIONS =  i18n/lumina-desktop_af.ts \
                 i18n/lumina-desktop_ar.ts \
@@ -169,7 +169,7 @@ TRANSLATIONS =  i18n/lumina-desktop_af.t
                 i18n/lumina-desktop_zh_TW.ts \
                 i18n/lumina-desktop_zu.ts
 
-dotrans.path=/usr/local/share/Lumina-DE/i18n/
-dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/local/share/Lumina-DE/i18n/
+dotrans.path=/usr/share/Lumina-DE/i18n/
+dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/share/Lumina-DE/i18n/
 
 INSTALLS += target desktop icons wallpapers defaults dotrans
--- lumina-desktop-0.6.2.14.orig/lumina-fm/lumina-fm.pro
+++ lumina-desktop-0.6.2.14/lumina-fm/lumina-fm.pro
@@ -2,7 +2,7 @@
 QT       += core gui phonon
 
 TARGET = lumina-fm
-target.path = /usr/local/bin
+target.path = /usr/bin
 
 TEMPLATE = app
 
@@ -22,7 +22,7 @@ FORMS    += MainUI.ui \
 
 # RESOURCES+= lumina-fm.qrc
 
-INCLUDEPATH += ../libLumina /usr/local/include
+INCLUDEPATH += ../libLumina /usr/include
 
 linux-g++-64 {
   LIBS     += -L../libLumina -lLuminaUtils 
@@ -95,10 +95,10 @@ TRANSLATIONS =  i18n/lumina-fm_af.ts \
                 i18n/lumina-fm_zh_TW.ts \
                 i18n/lumina-fm_zu.ts
 
-dotrans.path=/usr/local/share/Lumina-DE/i18n/
-dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/local/share/Lumina-DE/i18n/
+dotrans.path=/usr/share/Lumina-DE/i18n/
+dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/share/Lumina-DE/i18n/
 
 desktop.files=lumina-fm.desktop
-desktop.path=/usr/local/share/applications/
+desktop.path=/usr/share/applications/
 
 INSTALLS += target dotrans desktop
--- lumina-desktop-0.6.2.14.orig/lumina-open/lumina-open.pro
+++ lumina-desktop-0.6.2.14/lumina-open/lumina-open.pro
@@ -2,7 +2,7 @@
 QT       += core gui
 
 TARGET = lumina-open
-target.path = /usr/local/bin
+target.path = /usr/bin
 
 TEMPLATE = app
 
@@ -15,7 +15,7 @@ FORMS    += LFileDialog.ui
 
 RESOURCES+= lumina-open.qrc
 
-INCLUDEPATH += ../libLumina /usr/local/include
+INCLUDEPATH += ../libLumina /usr/include
 LIBS     += -L../libLumina -lLuminaUtils
 QMAKE_LIBDIR	= ../libLumina
 DEPENDPATH	+= ../libLumina
@@ -83,7 +83,7 @@ TRANSLATIONS =  i18n/lumina-open_af.ts \
                 i18n/lumina-open_zh_TW.ts \
                 i18n/lumina-open_zu.ts
 
-dotrans.path=/usr/local/share/Lumina-DE/i18n/
-dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/local/share/Lumina-DE/i18n/
+dotrans.path=/usr/share/Lumina-DE/i18n/
+dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/share/Lumina-DE/i18n/
 
 INSTALLS += target dotrans
--- lumina-desktop-0.6.2.14.orig/lumina-screenshot/lumina-screenshot.pro
+++ lumina-desktop-0.6.2.14/lumina-screenshot/lumina-screenshot.pro
@@ -2,7 +2,7 @@
 QT       += core gui
 
 TARGET = lumina-screenshot
-target.path = /usr/local/bin
+target.path = /usr/bin
 
 TEMPLATE = app
 
@@ -13,7 +13,7 @@ HEADERS  += MainUI.h
 
 FORMS    += MainUI.ui
 
-INCLUDEPATH += ../libLumina /usr/local/include
+INCLUDEPATH += ../libLumina /usr/include
 
 linux-g++-64 {
   LIBS     += -L../libLumina -lLuminaUtils 
@@ -86,10 +86,10 @@ TRANSLATIONS =  i18n/lumina-screenshot_a
                 i18n/lumina-screenshot_zh_TW.ts \
                 i18n/lumina-screenshot_zu.ts
 
-dotrans.path=/usr/local/share/Lumina-DE/i18n/
-dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/local/share/Lumina-DE/i18n/
+dotrans.path=/usr/share/Lumina-DE/i18n/
+dotrans.extra=cd i18n && lrelease-qt4 -nounfinished *.ts && cp *.qm $(INSTALL_ROOT)/usr/share/Lumina-DE/i18n/
 
 desktop.files=lumina-screenshot.desktop
-desktop.path=/usr/local/share/applications/
+desktop.path=/usr/share/applications/
 
 INSTALLS += target dotrans desktop
